src/components/CreateNewBook.css

```diff
@@ -0,0 +1,26 @@
+.book-form-title-style {
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.input-text {
+  height: 2.813rem;
+  width: 56%;
+  margin: 1.188rem 2.125rem 0 0;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.input-submit {
+  margin-bottom: 6px;
+  width: 15%;
+  height: 2.813rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+}
```

Comment:
**Changes Required**
The design looks good on a screen bigger than 1250px. However;
- [ ] The desktop media query size starts from 1024px. Let us please target starting from the smallest laptop computers, i.e 1024 and above.
Below you can see how it looks like on my laptop and what I mean.
![your-zeplin](https://user-images.githubusercontent.com/7220167/131222870-4d6be249-62b7-4fb8-9f7d-e2df44b7aee8.png)
![your-zeplin-form](https://user-images.githubusercontent.com/7220167/131222882-4d08fc57-f485-469c-9e52-8274a72f39d1.png)

Reply:
Hello, responsive design was not part of the requirements listed on the project milestone page. The only requirement listed was to make the page match the Zeplin design. My page currently matches the design required on the design mockup.